common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -4153,15 +4170,6 @@ object CelebornConf extends Logging {
       .booleanConf
       .createWithDefault(true)
 
-  val SPARK_SHUFFLE_FORCE_FALLBACK_ENABLED: ConfigEntry[Boolean] =
-    buildConf("celeborn.client.spark.shuffle.forceFallback.enabled")
```

Comment:
If you want to remove configuration items, it is recommended to add migration guidelines to `docs/migration.md`.

Reply:
done